packages/esm-form-entry-app/src/app/openmrs-api/encounter-resource.service.ts

```diff
@@ -66,7 +66,8 @@ export class EncounterResourceService {
       'visit:(uuid,visitType,display,startDatetime,stopDatetime),' +
       'location:ref,encounterType:ref,' +
       'encounterProviders:(uuid,display,provider:(uuid,display)),orders:full,' +
-      'obs:(uuid,obsDatetime,formFieldNamespace,formFieldPath,concept:(uuid,uuid,name:(display)),value:ref,groupMembers))';
+      'obs:(uuid,obsDatetime,formFieldNamespace,formFieldPath,concept:(uuid,uuid,name:(display)),value:ref,groupMembers),' +
+      'diagnoses:(uuid,diagnosis,certainty,rank,voided,display))';
```

Comment:
So, we should be able to get formFieldNamespace and formFieldPath for diagnoses too.

Reply:
Hello @ibacher,
I have done a request here I retrieve all diagnoses information and the `formFieldNamespace` and `formFieldPath` are not available. Please see below:

Request:
```
http://localhost:9090/openmrs/ws/rest/v1/encounter/063f1906-8ab9-4e27-ba75-1b001023d54d?v=custom:(uuid,encounterDatetime,patient:(uuid,uuid,person,identifiers:full),form:(uuid,name),visit:(uuid,visitType,display,startDatetime,stopDatetime),location:ref,encounterType:ref,encounterProviders:(uuid,display,provider:(uuid,display)),orders:full,obs:(uuid,obsDatetime,formFieldNamespace,formFieldPath,concept:(uuid,uuid,name:(display)),value:ref,groupMembers),diagnoses)
```

Reponse:
```
...
"diagnoses": [
        {
            "uuid": "4aac9625-ef9d-4250-83a3-6eaa8e6c7f4d",
            "diagnosis": {
                "coded": {
                    "uuid": "111539AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
                    "display": "Venous Blood Pressure Increase",
                    "links": [
                        {
                            "rel": "self",
                            "uri": "http://localhost:9090/openmrs/ws/rest/v1/concept/111539AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
                            "resourceAlias": "concept"
                        }
                    ]
                }
            },
            "condition": null,
            "encounter": {
                "uuid": "063f1906-8ab9-4e27-ba75-1b001023d54d",
                "display": "encounter.cries8Form 10/03/2023",
                "links": [
                    {
                        "rel": "self",
                        "uri": "http://localhost:9090/openmrs/ws/rest/v1/encounter/063f1906-8ab9-4e27-ba75-1b001023d54d",
                        "resourceAlias": "encounter"
                    }
                ]
            },
            "certainty": "CONFIRMED",
            "rank": 1,
            "voided": false,
            "display": "Venous Blood Pressure Increase",
            "patient": {
                "uuid": "86c70ec7-f133-4b79-aac1-d27e8103e878",
                "display": "JAL100546-100FVM - Joel Manel",
                "links": [
                    {
                        "rel": "self",
                        "uri": "http://localhost:9090/openmrs/ws/rest/v1/patient/86c70ec7-f133-4b79-aac1-d27e8103e878",
                        "resourceAlias": "patient"
                    }
                ]
            },
            "links": [
                {
                    "rel": "full",
                    "uri": "http://localhost:9090/openmrs/ws/rest/v1/patientdiagnoses/4aac9625-ef9d-4250-83a3-6eaa8e6c7f4d?v=full",
                    "resourceAlias": "patientdiagnoses"
                },
                {
                    "rel": "self",
                    "uri": "http://localhost:9090/openmrs/ws/rest/v1/patientdiagnoses/4aac9625-ef9d-4250-83a3-6eaa8e6c7f4d",
                    "resourceAlias": "patientdiagnoses"
                }
            ],
            "resourceVersion": "1.8"
        }
...
```